cli/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ _Released 12/17/2024 (PENDING)_
 **Bugfixes:**
 
 - Fixed an issue where targets may hang if `Network.enable` is not implemented for the target. Addresses [#29876](https://github.com/cypress-io/cypress/issues/29876).
+- Prevent overwriting of video files across multiple runs. Addresses [#8280](https://github.com/cypress-io/cypress/issues/8280). Addressed in [#30673](https://github.com/cypress-io/cypress/pull/30673).
```

Comment:
```suggestion
- Fixed an issue where the configuration setting `trashAssetsBeforeRuns=false` was ignored for assets in the `videosfolder` and these assets were incorrectly deleted before running tests with `cypress run`. Addresses [#8280](https://github.com/cypress-io/cypress/issues/8280).
```

1. Orient the Changelog text towards the configuration description in https://docs.cypress.io/app/references/configuration#Screenshots and use a similar style for the description as used in other Changelog entries
2. Only list the issue ID, not the PR ID as well (see https://github.com/cypress-io/cypress/blob/develop/guides/writing-the-cypress-changelog.md#writing-guidelines).